gcs-proxy.go

```diff
@@ -87,6 +87,7 @@ func (c *EncryptGcsPayload) Request(f *proxy.Flow) {
 
 	log.Debug(fmt.Sprintf("got request: %s", f.Request.Raw().RequestURI))
 	if IsEncryptDisabled() {
+		fmt.Println("Encryption Enabled: false")
```

Comment:
I already log this in [main.go](https://github.com/byronwhitlock-google/go-gcsproxy/blob/09fc2cf6e3e55ac0a23c43b7934393024b7e7316/main.go#L111). You probably don't want to print this for every request/response as the setting is per proxy.

Reply:
removed this debug lines.